pep-0678.rst

```diff
@@ -15,9 +15,9 @@ Abstract
 ========
 Exception objects are typically initialized with a message that describes the
 error which has occurred.  Because further information may be available when the
-exception is caught and re-raised, this PEP proposes to add a ``.__note__``
-attribute and update the builtin traceback formatting code to include it in
-the formatted traceback following the exception string.
+exception is caught and re-raised, or included in an ``ExceptionGroup`` , this PEP
+proposes to add a ``.__note__`` attribute and update the builtin traceback formatting
+code to include it in the formatted traceback following the exception string.
 
 This is particularly useful in relation to :pep:`654` ``ExceptionGroup`` s, which
```

Comment:
I can't test right now, but is this space required for Docutils to process the literal correctly? If it is, could you backslash escape it so that it renders properly?
```suggestion
This is particularly useful in relation to :pep:`654` ``ExceptionGroup``\ s, which
```

Reply:
> I can't test right now, but is this space required for Docutils to process the literal correctly?

Yes, unfortunately—in fact, it results in a hard error when parsing—but that's a good trick to avoid it.